spec/features/create_object_no_files_spec.rb

```diff
@@ -45,7 +45,7 @@
 
     # open a new version
     click_link 'Open for modification'
-    within '.modal-dialog' do
+    within '.modal-body' do
```

Comment:
I'm pretty sure the class we're looking for is `modal-dialog`. See https://github.com/sul-dlss/argo/blob/c3fef087978e09fe80770ce3af66538a505e704a/app/views/catalog/show.html.erb#L4

Reply:
The code on argo staging is currently in flux and there is a bug in the code. The test was looking correctly for modal-dialog.

Reply:
Got it. Should I just close this PR for now?

Reply:
Yes, I think so